bound/kt/src/main/kotlin/web5/sdk/crypto/signers/RustSignerAdapter.kt

```diff
@@ -0,0 +1,11 @@
+package web5.sdk.crypto.signers
+
+import web5.sdk.rust.SignerInterface as RustSignerInterface
+
+// TODO: Figure out if we need this
+class RustSignerAdapter(private val kotlinSigner: Signer) : RustSignerInterface {
```

Comment:
is this because of something weird UniFFI is code-gen'ing? not a show stopper just highlighting it in case you glossed over it prior to requesting review

Reply:
^ Yea I was going to give it another shot today, I think I need something like this to do diddht.publish()